numba/tests/test_np_randomgen.py

```diff
@@ -159,6 +163,44 @@ def test_bitgen_funcs(self):
             with self.subTest(_func=_func, _func_name=_func_name):
                 self._test_bitgen_func_parity(_func_name, _func)
 
+    def test_integers(self):
+        test_sizes = [None, (), (100,), (10, 20, 30)]
+        test_dtypes = [np.int64, np.int32, np.int16, np.int8]
```

Comment:
Would it be feasible to add the unsigned variants too?

Reply:
Added unsigned counterparts in latest commit.